invenio_openaire/indexer.py

```diff
@@ -45,7 +45,7 @@ def indexer_receiver(sender, json=None, record=None, index=None,
                 'funder': [json['funder']['doi']]
             },
             'payload': {
-                'id': json['internal_id']
+                'id': json['code']
```

Comment:
Passing two values in the payload doesn't necessarily help in the use case of the deposit form, since there should be one and only one place to look for an item's unique id/value (applying some sort of value resolving logic from the payload on the client side, can't really be contained inside a single module).
In the end what might work correctly is doing something like:
`'id': json['code'] if ...FP7... else json['internal_id']`